histomicsui/web_client/templates/panels/zoomWidget.pug

```diff
@@ -7,7 +7,7 @@ block content
   .input-group
     input.h-zoom-slider.form-control(
       type="range", min=min, max=max, step=step, value=value, disabled=disabled)
-    span.h-zoom-value.input-group-addon #{value}
+    input.h-zoom-value.input-group-addon(type="text", value=value)
```

Comment:
Does specifying `type="text"` do anything?  Isn't it the default?

Reply:
changed in cdc6fe78915a1822c076bfef9d8b4f15fbcc81bb